src/main/java/gregtech/api/mui/GTGuis.java

```diff
@@ -72,47 +72,81 @@ public static ModularPanel defaultPanel(MetaItem<?>.MetaValueItem valueItem) {
         return createPanel(valueItem.unlocalizedName);
     }
 
-    public static ModularPanel createPopupPanel(String name, int width, int height) {
-        return createPopupPanel(name, width, height, false, false);
+    public static PopupPanel createPopupPanel(String name, int width, int height) {
+        return defaultPopupPanel(name)
+                .size(width, height);
     }
 
-    public static ModularPanel createPopupPanel(String name, int width, int height, boolean disableBelow,
-                                                boolean closeOnOutsideClick) {
-        return new PopupPanel(name, width, height, disableBelow, closeOnOutsideClick);
+    public static PopupPanel createPopupPanel(String name, int width, int height, boolean deleteCachedPanel) {
+        return createPopupPanel(name, width, height)
+                .deleteCachedPanel(deleteCachedPanel);
     }
 
-    public static ModularPanel defaultPopupPanel(String name) {
-        return defaultPopupPanel(name, false, false);
+    public static PopupPanel defaultPopupPanel(String name) {
+        return new PopupPanel(name);
```

Comment:
To actually set the default size on `defaultPopupPanel`, since it was removed in the constructor.
Other than this, looks good.
```suggestion
        return new PopupPanel(name).size(DEFAULT_WIDTH, DEFAULT_HEIGHT);
```